cmd/validate.go

```diff
@@ -85,26 +86,26 @@ func (vc *validateCmd) run() error {
 	ctx := context.Background()
 	isDir, err := isDirectory(vc.manifestPath)
 	if err != nil {
-		return fmt.Errorf("could not determine if given path is a directory: %w", err)
+		return fmt.Errorf("not a valid file or directory: %w", err)
 	}
 
-	var manifests []string
+	var manifestFiles []string
 	var manifestFS = os.DirFS(vc.manifestPath)
```

Comment:
Since this is only used to pass into `getManifestFiles`, you can move the instantiation of manifestFS to inside getManifestFiles